app/components/buy-now.ts

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import type { ICellRendererParams } from 'ag-grid-community';
+
+
+type Args = {
+  params: ICellRendererParams
```

Comment:
Stronger typing could be use here : 
`params: ICellRendererParams<CustomRowDataDef>`